packages/components/core/styles/common/_tokens.scss

```
@@ -45,3 +45,31 @@ $tokens: meta.module-variables(tokens) !default;
     @include kbq-css-font-variable($component, $property, font-feature-settings, '');
     @include kbq-css-font-variable($component, $property, letter-spacing, '');
 }
+
+@function kbq-sum-css-variables($first, $second) {
+    @return calc(kbq-css-variable($first) + kbq-css-variable($second));
+}
+
+@function kbq-new-sum-css-variables($variables) {
```

Чтобы подчеркнуть возможность сложения нескольких чисел, можно добавить слово `series`
```suggestion
@function kbq-sum-series-css-variables($variables) {
```

---

new - то был просто первоначальный вариант, чтобы предложить использовать такую миксину)